Sources/StreamChatUI/ChatMessageList/ChatMessage/ChatMessageCell.swift

```diff
@@ -27,13 +35,23 @@ public final class ChatMessageCell: _TableViewCell {
     
     override public func setUpLayout() {
         super.setUpLayout()
+
+        containerStackView.axis = .vertical
+        containerStackView.alignment = .center
+        containerStackView.spacing = 8
+        containerStackView.addArrangedSubview(dateSeparatorView)
+        containerStackView.addArrangedSubview(messageContentView!)
+        contentView.addSubview(containerStackView)
         
-        contentView.addSubview(messageContentView!)
-        
-        messageContentView!.pin(
+        containerStackView.pin(
             anchors: [.leading, .top, .trailing, .bottom],
             to: contentView
         )
+
+        messageContentView!.pin(
```

Comment:
Can we do optional chaining here, please? 😄

Reply:
Yes sure, I just moved code in this case, but I also prefer to change it 👍

Reply:
Actually, I won't change this. The force unwrapping, in this case, is important to make sure we don't forget to set the message content view. So in order for this to crash, it is a developer mistake and it should be caught as early as possible.

Reply:
I tend to disagree with the "it is a developer mistake" statement. We can make mistakes, and we can even miss them in a release, using force unwraps does not provide safety to our libraries.

Reply:
@polqf Updated ✅